Why LightManager test fails ?

Hi all,

One test related to LightManager is failing only on windows eg:
https://github.com/sofa-framework/sofa/commit/6d3e1398363cc617124fd25522193ef20b1b7b1c

The failing scene is rather simple: 
```xml
<Node name='Root' gravity='0 -9.81 0' time='0' animate='0' >
    <Node name='Level 1'>
        <MechanicalObject template='Vec3d'/>
        <LightManager name='lightmanager'/>
    </Node>
</Node>
```
Any one know why the test on Windows behave differently to to other plateforms ? 

DM.

Some more info about this test...  
Fails on master since b70b477f
Error appeared in `defrost` branch between 4b1f289 and a33b4b1.
First CI build with error: https://ci.inria.fr/sofa-ci/job/windows7_VS-2015_default_amd64/546/  
Error message:  
```
J:\job\origin\master\modules\SofaOpenglVisual\SofaOpenglVisual_test\LightManager_test.cpp:65
Expected: (lm) != (nullptr), actual: NULL vs 8-byte object <00-00 00-00 00-00 00-00>
```
Full output:  
```
Running SofaOpenglVisual_test.exe
Seed Value used for the tests = 1478637948
[==========] Running 1 test from 1 test case.
[----------] Global test environment set-up.
[----------] 1 test from TestLightManager
[ RUN      ] TestLightManager.checkAttributes
[ERROR]   [DAGNode "Level 1"] Object type "MechanicalObject" creation Failed
                              Emitted from 'J:\job\origin\master\SofaKernel\modu
                              les\SofaSimulationCommon\xml\NodeElement.cpp' line 
                              92
[ERROR]   [DAGNode "Level 1"] Object type "LightManager" creation Failed
                              Emitted from 'J:\job\origin\master\SofaKernel\modu
                              les\SofaSimulationCommon\xml\NodeElement.cpp' line 
                              92
[ERROR]   [SceneLoaderXML] Node initialization failed.
                           Emitted from 'testscene' line 1
J:\job\origin\master\modules\SofaOpenglVisual\SofaOpenglVisual_test\LightManager_test.cpp(65): error: Expected: (lm) != (nullptr), actual: NULL vs 8-byte object <00-00 00-00 00-00 00-00>
unknown file: error: SEH exception with code 0xc0000005 thrown in the test body.
[  FAILED  ] TestLightManager.checkAttributes (9 ms)
[----------] 1 test from TestLightManager (9 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test case ran. (10 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] TestLightManager.checkAttributes
```

--- a/modules/SofaOpenglVisual/SofaOpenglVisual_test/LightManager_test.cpp
+++ b/modules/SofaOpenglVisual/SofaOpenglVisual_test/LightManager_test.cpp
@@ -4,7 +4,7 @@ using std::vector;
 #include <string>
 using std::string;
 
-#include <gtest/gtest.h>
+#include <SofaTest/Sofa_test.h>
 
 #include<sofa/core/objectmodel/BaseObject.h>
 using sofa::core::objectmodel::BaseObject ;
@@ -38,14 +38,11 @@ int messageInited = initMessage();
 
 namespace sofa {
 
-struct TestLightManager : public ::testing::Test {
-    void checkAttributes();
+struct TestLightManager : public Sofa_test<double> {
 };
 
-void TestLightManager::checkAttributes()
+void checkAttributes()
 {
-    sofa::simulation::setSimulation(new sofa::simulation::graph::DAGSimulation());
-
     std::stringstream scene ;
     scene << "<?xml version='1.0'?>"
              "<Node 	name='Root' gravity='0 -9.81 0' time='0' animate='0' >               \n"

